src/sql/src/plan/statement/ddl.rs

```diff
@@ -2138,6 +2140,7 @@ pub fn plan_create_sink(
     mut stmt: CreateSinkStatement<Aug>,
 ) -> Result<Plan, PlanError> {
     scx.require_unsafe_mode("CREATE SINK")?;
+    // XXX(chae): figure out what this should be if it's based on a source directly instead of a materialized view
```

Comment:
Why is the `compute_instance` even needed? (I asked above about this `index_oracle` thing)